### PR TITLE
ci(upgrades): fix dns-service ci access token

### PR DESCRIPTION
**Pull request by sreis**

### CODE REVIEW

**Comment by sreis** [comment]
Testing if it works in the upgrades pipeline [here](https://buildkite.com/opstrace/upgrade-tests/builds/855).

**Comment by sreis** [comment]
Needs the build artifacts from #929.

**Comment by sreis** [comment]
It's working [now](https://buildkite.com/opstrace/upgrade-tests/builds/859#472121bb-874b-4d00-834b-e1c4fa38b224/428-643
). Relevant log line showing it could talk to dns-service:

```
[2021-06-24T08:44:31Z] 2021-06-24T08:44:31.578Z debug: resp to PUT(https://dns-api.opstrace.net/dns/) -> 202. total time: 0.08 s. resp time: 0.05 s.
```

File: ci/test-upgrade/run.sh [hunk 1]
```diff
@@ -5,7 +5,10 @@ set -eou pipefail
 make ci-fetch-secrets
 
 make ci-testupgrade-fetch-cli-artifacts
-make ci-testupgrade-dns-service-credentials
+
+echo "--- setting up dns-service credentials"
+# The `access.jwt` file is what the CLI is going to look for.
+cp secrets/dns-service-magic-id-token-for-ci access.jwt
```
Comment (jgehrcke) [inline ci/test-upgrade/run.sh:11]: Thanks @sreis and sorry for not thinking of that!

**Pull request merged**